### PR TITLE
WRN-1352: Migrated tests for Picker.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated tests from Enzyme to Testing Library.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed test structure to work with Testing Library.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1352

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: Picker/tests/Picker-specs.js [hunk 1]
```diff
@@ -1,100 +1,105 @@
...
+		const picker = getByTestId('picker');
 		const expected = '2';
-		const actual = picker.find('PickerItem').text();
+		const actual = picker.children.item(1).textContent;
 
 		expect(actual).toBe(expected);
 	});
 
 	test(
 		'should set the max of <Picker> to be one less than the number of children',
```
Comment (stanca-pop-lgp) [inline Picker/tests/Picker-specs.js:22]: Maybe add this on a single line like the other test descriptions.

File: Picker/tests/Picker-specs.js [hunk 2]
```diff
@@ -1,100 +1,105 @@
...
-		const expected = true;
-		const actual = picker.find('PickerButton').at(1).prop('data-webos-voice-disabled');
+		const picker = getAllByRole('button')[0];
+		const expected = 'data-webos-voice-disabled';
 
-		expect(actual).toBe(expected);
+		expect(picker).toHaveAttribute(expected);
 	});
 
 	test('should have an heading element when \'title\'', () => {
```
Comment (stanca-pop-lgp) [inline Picker/tests/Picker-specs.js:77]: Typo: "a" heading.

File: Picker/tests/Picker-specs.js (hunk 1 above)
Comment (alexandrumorariu) [reply Picker/tests/Picker-specs.js:22]: Ok

File: Picker/tests/Picker-specs.js (hunk 2 above)
Comment (alexandrumorariu) [reply Picker/tests/Picker-specs.js:75]: Will fix.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**